docs/_includes/expressive-loader.html

```diff
@@ -142,11 +142,15 @@ <h3 id="expressive-loader-messages-no-reduced-motion">
   </div>
   {% endhighlight %}
 
-  <h4>
-    Multiple messages (default behavior)
-  </h4>
+  <h3 id="expressive-loader-multiple-messages">
+    Expressive Loader with Multiple Messages
+  </h3>
+  <p>
+    For the following examples, we're using Javascript to rotate through messages.
```

Comment:
We should be explicit that this `js` is not included in Skin. We do this throughout Skin docs. here's one instance:

<img width="1300" alt="image" src="https://github.com/eBay/skin/assets/1675667/d514440e-03b9-4c9d-b38d-503d173def52">